lib/fluent/plugin/in_cloudwatch_logs.rb

```diff
@@ -144,8 +144,13 @@ def emit(stream, event)
           router.emit(@tag, time, record)
         }
       else
+        begin
+          record = @json_handler.load(event.message)
+        rescue JSON::ParserError, Yajl::ParseError # Catch parser errors
+          log.debug "Non-JSON message encountered"
+          record = { message: event.message }
+        end
```

Comment:
This comment is not against but my curiosity.

In this case, I want to use `#emit_error_event(tag, time, record, error)` instead of just re-formatting with message key.

WDYT?

Reply:
That's probably a more canonical way to handle non-JSON messages. I'll update the PR.